asset-properties-widget/component/asset-properties-config/asset-property-selector/asset-property-selector.component.ts

```diff
@@ -44,7 +44,7 @@ export class AssetPropertiesSelectorComponent {
   ) {}
 
   async ngOnChanges(changes: IManagedObject): Promise<void> {
-    if (changes.asset.firstChange && this.config?.properties) {
+    if ((changes.asset.firstChange && this.config?.properties) || (!changes.asset.previousValue && this.config?.properties)) {
```

Comment:
This can be simplified as : `if ((changes.asset.firstChange || !changes.asset.previousValue) && this.config?.properties)`

Reply:
incorporated